src/memory/memory_shared.h

```diff
@@ -82,6 +86,10 @@ typedef union {
         uint8_t encryption_key_split[32];
         uint8_t platform;
         uint8_t reserved[3]; // align to 4 bytes
+        uint8_t ble_bond_db[MEMORY_BLE_BOND_DB_LEN]; // 2 bytes len, then data
```

Comment:
should split out ble_bond_db in `uint16_t ble_bond_db_len`.

Could also reduce `MEMORY_BLE_BOND_DB_LEN` with 2 to avoid padding later.